### PR TITLE
add docs for PR 33302

### LINKED ISSUE #44791: need docs on s3 etc. see pr.
docs in https://github.com/ClickHouse/ClickHouse/pull/33302

### PR DESCRIPTION
**Pull request by DanRoscigno**
closes #44791 

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/operations/settings/settings.md [hunk 1]
```diff
@@ -5227,3 +5235,34 @@ Disabled by default.
 The maximum allowed size for String in RowBinary format. It prevents allocating large amount of memory in case of corrupted data. 0 means there is no limit.
 
 Default value: `1GiB`
+
+## S3 settings
```
Comment (Avogar) [inline docs/en/operations/settings/settings.md:5239]: That's not a good place for these settings. This section at the end of this doc is all about Format settings (there is a header `# Format settings`). Let's add these settings before format settings (as setting `engine_file_allow_create_multiple_files`), so we won't mix them up.
BTW, what do you think about transferring format settings into a separate file? In general this doc is really big and almost has no structure (maybe you have ideas how we can improve it).

Comment (Avogar) [reply docs/en/operations/settings/settings.md:5239]: Hmm, do you know why we don't have `Format settings` section in the right section?
Screenshot 2023-01-23 at 22 55 56

Comment (DanRoscigno) [reply docs/en/operations/settings/settings.md:5239]: Maybe the parser gave up after 5,000 lines :(

Comment (DanRoscigno) [reply docs/en/operations/settings/settings.md:5239]: > That's not a good place for these settings. This section at the end of this doc is all about Format settings (there is a header `# Format settings`). Let's add these settings before format settings (as setting `engine_file_allow_create_multiple_files`), so we won't mix them up. BTW, what do you think about transferring format settings into a separate file? In general this doc is really big and almost has no structure (maybe you have ideas how we can improve it).

I will move format settings to another file.

Are there other groups that can move out?

Comment (Avogar) [reply docs/en/operations/settings/settings.md:5239]: AFAIK no other groups. Some time ago format settings were mixed with other settings and I spent some time reorganazing them.

Comment (Avogar) [reply docs/en/operations/settings/settings.md:5239]: About `Format settings` header, just checked other placed in out docs and seems like all headers with single `#` are not presented in the right section.
Do you know how to fix it? Or maybe I should ask someone else

Comment (DanRoscigno) [reply docs/en/operations/settings/settings.md:5239]: > In the `md` file it's `# Format settings {#format-settings}`

Ah, there can only be one level 1 heading in the file, so it got skipped.

Comment (DanRoscigno) [reply docs/en/operations/settings/settings.md:5239]: I am going to have to reorganize this page, there are join settings in here and also in the join docs, merge_tree settings in here and also in the merge_tree docs, etc.  I will come back to this tomorrow.

File: docs/en/operations/settings/settings.md [hunk 2]
```diff
@@ -3707,7 +3767,7 @@ Possible values:
 
 Default value: 2.
 
-## compatibility {#compatibility}
+### compatibility {#compatibility}
```
Comment (Avogar) [inline docs/en/operations/settings/settings.md:3770]: This setting and `allow_settings_after_format_in_insert` are not related to `Schema inference settings`

Comment (DanRoscigno) [inline docs/en/operations/settings/settings.md:3770]:
```suggestion
## compatibility {#compatibility}
```

File: docs/en/operations/settings/settings.md [hunk 3]
```diff
@@ -3721,7 +3781,7 @@ Disabled by default.
 In ClickHouse Cloud the compatibility setting must be set by ClickHouse Cloud support.  Please [open a case](https://clickhouse.cloud/support) to have it set.
 :::
 
-## allow_settings_after_format_in_insert {#allow_settings_after_format_in_insert}
+### allow_settings_after_format_in_insert {#allow_settings_after_format_in_insert}
```
Comment (DanRoscigno) [inline docs/en/operations/settings/settings.md:3784]:
```suggestion
## allow_settings_after_format_in_insert {#allow_settings_after_format_in_insert}
```

**Pull request merged**